Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM docker/buildx-bin:$BUILDX_VERSION as buildx_bin
 FROM docker:latest as buildx_image
 
 ARG DOCKER_CONFIG=/env_configs/.docker
+ARG BUILDX_VERSION
```

Comment:
Now you now why I had that line in my PR in the first place... ;-)

Reply:
Ha.

Mayyybr